src/main/java/roomescape/dto/ReservationSaveRequest.java

```diff
@@ -0,0 +1,17 @@
+package roomescape.dto;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import java.time.LocalDate;
+import roomescape.domain.Reservation;
+import roomescape.domain.ReservationTime;
+
+public record ReservationSaveRequest(
+        String name,
+        LocalDate date,
+        @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "HH:mm")
+        Long timeId) {
```

Comment:
요것도 취향차이이지만 아래처럼 개행해주는것이 가독성에서 더 좋다고 생각해요!

```suggestion
        @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "HH:mm")
        Long timeId
) {
```